examples/run_nutation.py

```diff
@@ -79,20 +80,18 @@
         repetition_us=config_dict["repetition_us"],
         tau_us=config_dict["tau_us"],
         SW_kHz=config_dict["SW_kHz"],
-        ret_data=nutation_data,
+        ret_data=data,
     )
-nutation_data.setaxis("indirect", p90_range_us * 1e-6).set_units(
-    "indirect", "s"
-)
+data.setaxis("indirect", p90_range_us * 1e-6).set_units("indirect", "s")
 # {{{ chunk and save data
-nutation_data.chunk("t", ["ph1", "t2"], [4, -1])
-nutation_data.setaxis("ph1", ph1_cyc / 4)
+data.chunk("t", ["ph1", "t2"], [4, -1])
+data.setaxis("ph1", ph1_cyc / 4)
 if config_dict["nScans"] > 1:
-    nutation_data.setaxis("nScans", r_[0 : config_dict["nScans"]])
-nutation_data.reorder(["ph1", "nScans", "t2"])
-nutation_data.set_units("t2", "s")
-nutation_data.set_prop("postproc_type", "spincore_nutation_v4")
-nutation_data.set_prop("coherence_pathway", {"ph1": +1})
-nutation_data.set_prop("acq_params", config_dict.asdict())
+    data.setaxis("nScans", r_[0 : config_dict["nScans"]])
+data.reorder(["ph1", "nScans", "t2"])
+data.set_units("t2", "s")
+data.set_prop("postproc_type", "spincore_nutation_v4")
+data.set_prop("coherence_pathway", {"ph1": +1})
+data.set_prop("acq_params", config_dict.asdict())
```

Comment:
while it's good to try to limit the diff, I notice that we keep having problems with ppgs, where the `data` variable is named differently, so I fix that.